pyxem/signals/reduced_intensity_profile.py

```diff
@@ -0,0 +1,168 @@
+        s_max : float
+                    The maximum s value to be used for transformation to PDF.
+        """
+        s_scale = self.axes_manager.signal_axes[0].scale
+        s_size = self.axes_manager.signal_axes[0].size
+        if not s_max:
+            s_max = s_scale * s_size
+        delta = np.pi / s_max
+
+        scattering_axis = s_scale * np.arange(s_size, dtype='float64')
+        damping_term = np.sin(delta * scattering_axis) / (delta * scattering_axis)
+        damping_term = np.nan_to_num(damping_term)
+        self.data = self.data * damping_term
+        return
+
+    def damp_updated_lorch(self, s_max=None):
+        """ Damps the reduced intensity signal to reduce noise in the high s
+        region by a factor of 3 / (s*delta)^3 (sin(s*delta)-s*delta(cos(s*delta))),
+        where delta = pi / s_max.
+        From "Extracting the pair distribution function from white-beam X-ray
+        total scattering data", Soper & Barney, 2011
+
+        Parameters
+        ----------
+        s_max : float
+                    the damping parameter, which need not be the maximum s
+                    to be used for the PDF transform. It's a good guess however
+        """
+        s_scale = self.axes_manager.signal_axes[0].scale
+        s_size = self.axes_manager.signal_axes[0].size
+        if not s_max:
+            s_max = s_scale * s_size
+        delta = np.pi / s_max
+
+        scattering_axis = s_scale * np.arange(s_size, dtype='float64')
+        exponent_array = 3 * np.ones(scattering_axis.shape)
+        cubic_array = np.power(scattering_axis, exponent_array)
+        multiplicative_term = np.divide(3 / (delta**3), cubic_array)
+        sine_term = (np.sin(delta * scattering_axis)
+                     - delta * scattering_axis * np.cos(delta * scattering_axis))
+
+        damping_term = multiplicative_term * sine_term
+        damping_term = np.nan_to_num(damping_term)
+        self.data = self.data * damping_term
+        return
+
+    def damp_low_q_region_erfc(self, scale=20, offset=1.3):
+        """ Damps the reduced intensity signal in the low q region as a
+        correction to central beam effects. The reduced intensity profile is
+        damped by (erf(scale * s - offset) + 1) / 2
+
+        Parameters
+        ----------
+        scale : float
+                    A scalar multiplier for s in the error function
+        offset : float
+                    A scalar offset affecting the error function.
+        """
+        s_scale = self.axes_manager.signal_axes[0].scale
+        s_size = self.axes_manager.signal_axes[0].size
+
+        scattering_axis = s_scale * np.arange(s_size, dtype='float64')
+
+        damping_term = (special.erf(scattering_axis * scale - offset) + 1) / 2
+        self.data = self.data * damping_term
+        return
+
+    def fit_thermal_multiple_scattering_correction(self, s_max=None, plot=False):
+        """ Fits a 4th order polynomial function to the reduced intensity.
```

Comment:
I'm not totally sure if this should be done here - or rather done in workflow in a demo notebook... @pc494 do you have an opinion?